server/db/mgo.go

```diff
@@ -713,3 +716,100 @@ func (m *mongoDB) GetAddressInfo(ctx context.Context, hash string) (*types.Addre
 }
 
 //endregion Address
+
+// start region Proposal
+
+func (m *mongoDB) AddVoteToProposal(ctx context.Context, proposalInfo *types.ProposalDetail, voteOption uint64) error {
+	currentProposal, err := m.ProposalInfo(ctx, proposalInfo.ID)
+	if err != nil {
+		currentProposal = proposalInfo
+	}
+	// update number of vote choices
+	switch voteOption {
+	case 0 :
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)